warehouse/models/intermediate/gtfs_quality/guidelines_checks/int_gtfs_quality__feed_listed_sa.sql

```diff
@@ -0,0 +1,38 @@
+WITH
+
+idx AS (
+    SELECT * FROM {{ ref('int_gtfs_quality__services_guideline_index') }}
+),
+
+dim_provider_gtfs_data AS (
+    SELECT * FROM {{ ref('dim_provider_gtfs_data') }}
+),
+
+joined AS (
+    SELECT
+        idx.date,
+        idx.service_key,
+        -- Some services are mapped to multiple service_alerts_gtfs_dataset_key values at the same time.
+        -- This is one way to address that, open to better solutions!
+        ANY_VALUE(service_alerts_gtfs_dataset_key) AS gtfs_dataset_key
+    FROM idx
+    LEFT JOIN dim_provider_gtfs_data AS quartet
+        ON idx.service_key = quartet.service_key
+       AND TIMESTAMP(idx.date) BETWEEN quartet._valid_from AND quartet._valid_to
+       AND quartet.guidelines_assessed
```

Comment:
So, this makes me a bit nervous, because of how the guidelines automation & the reports site are using different definitions of "assessed". This definition means we are making a service-level assessment based on guidelines-assessed data which may not be selected in the reports site. In practice I think this will largely be fine because of how the MTC 511 data is structured, but can we add a `TODO` comment here indicating that this might be fragile when the reports site eventually starts surfacing this data? 

I can also see if in the reindexing work I can think of a different way to handle 🤔

Reply:
Since we're using the ANY_VALUE() function, this logic is pretty forgiving. If it makes you more comfortable, we can remove the guidelines_assessed filter and it will pretty much work the same. What do you think?

Reply:
Let's maybe just replace with `where customer_facing` instead? More forgiving but I don't want to give people credit if we have access to private feeds for example